externs/managedmediasource.js

```diff
@@ -0,0 +1,78 @@
+/*! @license
+ * Shaka Player
+ * Copyright 2016 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Externs for ManagedMediaSource which were missing in the
+ * Closure compiler.
+ *
+ * @externs
+ */
+
+/**
+ * @constructor
+ * @implements {EventTarget}
```

Comment:
Couldn't be `@implements/extends {MediaSource}` to make this file shorter?

Reply:
I've tried, but the compiler always gives me problems with this...

Reply:
Thanks to @theodab, I managed to simplify the file!